packages/beacon-node/src/node/nodejs.ts

```diff
@@ -316,11 +323,11 @@ export class BeaconNode {
       if (this.metricsServer) await this.metricsServer.stop();
       if (this.monitoring) this.monitoring.stop();
       if (this.restApi) await this.restApi.close();
-
       await this.chain.persistToDisk();
       await this.chain.close();
-      await this.db.stop();
       if (this.controller) this.controller.abort();
```

Comment:
Abort must be called before closing db

Reply:
In general, it is best to abort as late as possible in shutdown sequence. This ensures all intervals/timeouts are cleared first and then all remaining pending operations are aborted. Prevents edge case where abort is done and interval schedules another operation which would be stuck pending and delay shutdown.